src/github.com/t3chguy/matrix-static/templates/room-chat.qtpl

```diff
@@ -199,7 +199,7 @@
         {% if memberInfo.AvatarURL.IsValid() %}
             <img class="avatar userAvatar" src="{%s memberInfo.AvatarURL.ToThumbURL(48, 48, "crop") %}" />
         {% else %}
-            <img class="avatar userAvatar" src="./img/logo_missing.png" />
+            <img class="avatar userAvatar" src="./avatar/{%u memberInfo.GetName() %}" />
```

Comment:
Have you tested this against directory escape attacks? E.g if I have an name of `../../../../etc/passwd` Gin doesn't try to resolve that right? I don't think it will, but I would like to check with you first.